### PR TITLE
Crediblee test

### PR DESCRIPTION
**Pull request by jblanava**
add the i18next translations and dropdown language selector button that keeps the selected language between screens.

### CODE REVIEW

**Comment by DionisioRD** [comment]
> add the i18next translations and dropdown language selector button that keeps the selected language between screens.

Can You add some screenshots of the dropdowns to see how they ended up looking?

**Comment by jblanava** [comment]
Screenshot added in the root directory

**Comment by DionisioRD** [comment]
> Screenshot added in the root directory

Well, we don't need the screenshots in the code, sorry.

What I meant was to add them to the PR so the reviewers have more context about the changes :laughing:

File: client/.env [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
+REACT_APP_LOCIZE_REFLNG = en
+REACT_APP_LOCIZE_VERSION = latest
+
+
+
+
+# REACT_APP_LOCIZE_PROJECTID = b660bfc5-52a5-47a8-ace9-37cb21c65d91
+# REACT_APP_LOCIZE_APIKEY = 68b4a710-6acb-4fc4-9f9a-396c47f3d102
+# REACT_APP_LOCIZE_REFLNG = en
+# REACT_APP_LOCIZE_VERSION = latest
```
Comment (DionisioRD) [inline client/.env]: Do we need these comments?

Also, add an empty line at the EOF.

File: crediblee-test/index.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+
```
Comment (DionisioRD) [inline crediblee-test/index.js:1]: What is all of this crediblee-test stuff?

Is it related to the project?

**Comment by jblanava** [comment]
Dropdown language selector screenshot

image

File: client/.env (hunk 1 above)
Comment (jblanava) [reply client/.env]: Cleaned.

File: crediblee-test/index.js (hunk 2 above)
Comment (jblanava) [reply crediblee-test/index.js:1]: Well, it was a internationalization test for a specific translation in one of the crediblee template. It's a different project with it's own package.json and when executed it shows the template with internationalization implemented.

Comment (jblanava) [reply crediblee-test/index.js:1]: But overall i think is more of a test feature for crediblee instead of a logic addition to this project :)

Comment (DionisioRD) [reply crediblee-test/index.js:1]: Do you think it should be then inside this repo? I don't see any relation with the current project.

Comment (jblanava) [reply crediblee-test/index.js:1]: You right

**Pull request closed without merging**